src/main/kotlin/lotto/WinningTicket.kt

```diff
@@ -0,0 +1,11 @@
+package lotto
+
+class WinningTicket(val winningNumbers: List<Int>, val bonusNumber: Int = 1) {
```

Comment:
Consider wrapping the collection (e.g., `winningNumbers`) in a value object as well.